src/store/holochain.js

```diff
@@ -223,10 +249,27 @@ export default {
           }
         }
       }, 1000)
+
+      // timeout initalize client attempt loop at 30min
```

Comment:
I didn't understand this comment because timeout is not really a verb in my book.

```suggestion
      // give up after 30 mins
```